examples/serverside-sorting/index.js

```diff
@@ -0,0 +1,215 @@
+        ['Terry Macdonald', 'Commercial Specialist', 'Miami', 39, '$140,000'],
+        ['Justice Mccarthy', 'Attorney', 'Tucson', 26, '$330,000'],
+        ['Silver Carey', 'Computer Scientist', 'Memphis', 47, '$250,000'],
+        ['Franky Miles', 'Industrial Analyst', 'Buffalo', 49, '$190,000'],
+        ['Glen Nixon', 'Corporate Counselor', 'Arlington', 44, '$80,000'],
+      ];
+
+      var offset = page * 10;
+      var data = [];
+
+      if (order !== '') {
+        if (order === 'asc') {
+          var tempData = srcData.sort((a, b) => {
+            return a[3] - b[3];
+          });
+
+          data =
+            offset + 10 >= srcData.length
+              ? tempData.slice(offset, srcData.length)
+              : tempData.slice(offset, offset + 10);
+        } else {
+          tempData = srcData.sort((a, b) => {
+            return b[3] - a[3];
+          });
+
+          data =
+            offset + 10 >= srcData.length
+              ? tempData.slice(offset, srcData.length)
+              : tempData.slice(offset, offset + 10);
+        }
+      } else {
+        data =
+          offset + 10 >= srcData.length ? srcData.slice(offset, srcData.length) : srcData.slice(offset, offset + 10);
+      }
+
+      setTimeout(() => {
+        resolve(data);
+      }, 250);
+    });
+  };
+
+  sort = (column, order) => {
+    let temp = {};
+    temp.column = column;
+    temp.order = order;
+    temp.page = this.state.page;
+
+    let newColumnSortDirections = ['none', 'none', 'none', 'none', 'none'];
+
+    switch (column) {
+      case 'Name':
+        newColumnSortDirections[0] = order;
+        break;
+      case 'Title':
+        newColumnSortDirections[1] = order;
+        break;
+      case 'Location':
+        newColumnSortDirections[2] = order;
+        break;
+      case 'Age':
+        newColumnSortDirections[3] = order;
+        break;
+      case 'Salary':
+        newColumnSortDirections[4] = order;
+        break;
+      default:
+        break;
+    }
+
+    newColumnSortDirections[column.index] = order;
+
+    this.xhrRequest(temp).then(data => {
+      this.setState({
+        data,
+        columnSortDirection: newColumnSortDirections,
+      });
+    });
+  };
+
+  render() {
+    const columns = [
+      {
+        name: 'Name',
+        options: {
+          sortDirection: this.state.columnSortDirection[0],
+          customFilterListRender: v => `Name: ${v}`,
+        },
+      },
+      {
+        name: 'Title',
+        options: {
+          sortDirection: this.state.columnSortDirection[1],
+          customFilterListRender: v => `Title: ${v}`,
+        },
+      },
+      {
+        name: 'Location',
+        options: {
+          sortDirection: this.state.columnSortDirection[2],
+          customFilterListRender: v => `Location: ${v}`,
+          customBodyRender: (value, tableMeta, updateValue) => {
+            return <Cities value={value} index={tableMeta.columnIndex} change={event => updateValue(event)} />;
```

Comment:
Since the initial data is just a loading message, we're missing `value` at first, which is causing a prop check verification error in the console. We could fix this by using `value || ''`.